chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -562,16 +563,33 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
     const confirmationPaneProps = initConfirmationPropsComposer(props);
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const prepareEndChatRelay = () => prepareEndChat(props, chatSDK, state, dispatch, setAdapter, setWebChatStyles, adapter, uwid.current);
-    
+
     const downloadTranscriptProps = createDownloadTranscriptProps(props.downloadTranscriptProps as IDownloadTranscriptProps,
         {
             ...defaultWebChatContainerStatefulProps?.webChatStyles,
             ...props.webChatContainerProps?.webChatStyles
         },
         props.webChatContainerProps);
-       
+
     const livechatProps = {...props, downloadTranscriptProps};
 
+    const chatWidgetDraggableConfig = {
+        elementId: widgetElementId,
+        channel: props.controlProps?.widgetInstanceId ?? "lcw",
+        disabled: props.draggableChatWidgetProps?.disabled === true ?? false // Draggable by default, unless explicitly disabled
```

Comment:
Are we sure we want to enable this functionality off the bat? Perhaps it should be the other way around, where customers can enable draggability if desired, since it introduces less risk/change to them. Customers who don't want this additional functionality don't need to do anything, whereas customers who think this new feature is beneficial can make changes to consume it.

Reply:
If we enable by default, do we need to send communications to customers so they are aware of this new UX change?

Reply:
It was discussed offline to have it enabled by default